src/General/MB.java

```diff
@@ -79,4 +83,20 @@ public static void show(MBPanel panel) {
         panel.afterVisible();
     }
 
+    /**
+     * Load a image file from the rsc folder via directory/name.
+     *
+     * @param relativePath
+     * @return
+     */
+    public static BufferedImage load(String relativePath){
+        System.out.println(System.getProperty("user.dir"));
```

Comment:
The method's parameters aren't documented and the method still contains a console output.
-> Describe the parameters and remove the print